lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -1281,8 +1281,11 @@ spec = do
         targetWallet <- emptyWallet ctx
 
         -- migrate funds and quickly get id of one of the pending txs
-        let payload = Json [json|{"passphrase": #{fixturePassphrase}}|]
-        let migrEp = Link.migrateWallet sourceWallet targetWallet
+        addrs <- listAddresses @n ctx targetWallet
+        let addr1 = (addrs !! 1) ^. #id
+
+        let payload = Json [json|{"passphrase": #{fixturePassphrase}, addresses: [#{addr1}]}|]
```

Comment:
```suggestion
        let payload = Json [json|
                { passphrase: #{fixturePassphrase}
                , addresses: [#{addr1}]
                }|]
```

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1664/commits/cf2b9e58c71fd7e80f8a9e1b9073348dfba3a67e